src/main/webapp/utilities/MapsInterface.js

```diff
@@ -0,0 +1,105 @@
+// TODO(tzavidas): seperate the interface from the logic, by using an implementation for easier testing (like the bridge pattern)
+class MapInterface {
+    /**
+     * @param element the DOM element the map to be rendered
+     * @param centerCoordinates coordinates object that will be the center of the map
+     * @param options (optional) additional parameters that might be needed
```

Comment:
I think this should be

     * @param {zoom: number, directionsService: type, directionsRenderer: type} options (optional) additional parameters that might be needed

https://google.github.io/styleguide/jsguide.html#jsdoc-type-annotations